src/test/java/bookface/testutil/TypicalBooks.java

```diff
@@ -16,19 +15,19 @@
  */
 public class TypicalBooks {
 
-    public static final Book HOWTOSPELL = new BookBuilder().withTitle("How to Spell")
+    public static final Book HOW_TO_SPELL = new BookBuilder().withTitle("How to Spell")
             .withAuthor("Ronald Johnson").build();
-    public static final Book MAKINGACOMPUTER = new BookBuilder().withTitle("Making a Computer")
+    public static final Book MAKING_A_COMPUTER = new BookBuilder().withTitle("Making a Computer")
             .withAuthor("Roger Smith").build();
-    public static final Book GETMOTIVATED = new BookBuilder().withTitle("Get Motivated")
+    public static final Book GET_MOTIVATED = new BookBuilder().withTitle("Get Motivated")
             .withAuthor("Lim Chee Teck").build();
 
 
     // Manually added - Book's details found in {@code CommandTestUtil}
-    public static final Book HowToSpell = new BookBuilder().withTitle(CommandTestUtil.VALID_TITLE_HOW)
-            .withAuthor(CommandTestUtil.VALID_AUTHOR_RONALD).build();
-    public static final Book MakingAComputer = new BookBuilder().withTitle(CommandTestUtil.VALID_TITLE_MAKING)
-            .withAuthor(CommandTestUtil.VALID_AUTHOR_ROGER).build();
+    //    public static final Book HowToSpell = new BookBuilder().withTitle(CommandTestUtil.VALID_TITLE_HOW)
+    //            .withAuthor(CommandTestUtil.VALID_AUTHOR_RONALD).build();
+    //    public static final Book MakingAComputer = new BookBuilder().withTitle(CommandTestUtil.VALID_TITLE_MAKING)
+    //            .withAuthor(CommandTestUtil.VALID_AUTHOR_ROGER).build();
```

Comment:
Perhaps we can remove this code if it does not value-add to the test anymore.
But if it is meant to be fixed (i.e: they currently fail), then we can make a note to come back to them in the future.

Reply:
Yep good point, I'll remove it